### PR TITLE
nginx: fix formating the config file

### PR DESCRIPTION
**Pull request by Izorkin**
###### Motivation for this change
Fix this PR #56280

cc @etu 

###### Things done

- [x] Tested using sandboxing ([nix.useSandbox](http://nixos.org/nixos/manual/options.html#opt-nix.useSandbox) on NixOS, or option `sandbox` in [`nix.conf`](http://nixos.org/nix/manual/#sec-conf-file) on non-NixOS)
- Built on platform(s)
   - [x] NixOS
   - [ ] macOS
   - [ ] other Linux distributions
- [ ] Tested via one or more NixOS test(s) if existing and applicable for the change (look inside [nixos/tests](https://github.com/NixOS/nixpkgs/blob/master/nixos/tests))
- [ ] Tested compilation of all pkgs that depend on this change using `nix-shell -p nox --run "nox-review wip"`
- [x] Tested execution of all binary files (usually in `./result/bin/`)
- [ ] Determined the impact on package closure size (by running `nix path-info -S` before and after)
- [ ] Assured whether relevant documentation is up to date
- [ ] Fits [CONTRIBUTING.md](https://github.com/NixOS/nixpkgs/blob/master/.github/CONTRIBUTING.md).

---

### CODE REVIEW

**Comment by etu** [comment]
@GrahamcOfBorg test nginx

**Comment by Izorkin** [comment]
Thanks!

File: nixos/modules/services/web-servers/nginx/default.nix [hunk 1]
```diff
@@ -44,23 +44,17 @@ let
...
-  configFile = pkgs.stdenv.mkDerivation {
-    name = "nginx-config";
-    src = "";
-    phases = [ "installPhase" ];
-    installPhase = ''
-      mkdir $out
-      awk -f ${awkFormat} ${pre-configFile} | sed '/^\s*$/d' > $out/nginx.conf
-    '';
-  };
+  configFile = pkgs.runCommand "nginx.conf" {} (''
```
Comment (Infinisil) [inline nixos/modules/services/web-servers/nginx/default.nix]: These parenthesis aren't needed

**Review by infinisil — COMMENTED** [review]
When you open a new PR to fix things I mentioned in a review, you should ping me, there's still something that could have been done better..

**Comment by Infinisil** [comment]
Also format your commit message according to the [Contribution Guidelines](https://github.com/NixOS/nixpkgs/blob/master/.github/CONTRIBUTING.md#submitting-changes), also linked to from the PR template in the future

**Pull request merged**